app/src/main/java/rkr/simplekeyboard/inputmethod/latin/utils/SubtypeLocaleUtils.java

```diff
@@ -386,8 +387,14 @@ public List<Subtype> getSubtypes() {
                 case LOCALE_MALAY_MALAYSIA:
                 case LOCALE_DUTCH:
                 case LOCALE_POLISH:
-                case LOCALE_PORTUGUESE_BRAZIL:
+                case LOCALE_PORTUGUESE_BRAZIL:				
```

Comment:
This switch statement is very wrong. `LAYOUT_HCESAR` will be included in all the locales listed above because the way it works is it finds the `mLocale` line and then executes all the code below till the first `break` statement. Ti fix this you will need to move `LOCALE_PORTUGUESE_BRAZIL` and `LOCALE_PORTUGUESE_PORTUGAL` into a separate group and then `break`.

Reply:
Me too, but your code adds them to all languages now :)